docs/src/index.md

```diff
@@ -11,7 +17,7 @@ RayTracer.jl is a library for differentiable ray tracing. It provides utilities
 Download [Julia 1.1](https://julialang.org/) or later. 
 
 !!! note
-    This library won't work with Julia 1.0.
+    This library won't work with Julia 1.0 as it needs the `isnothing` function.
```

Comment:
Two solutions: either you use `=== nothing` or Compat.jl (https://github.com/JuliaLang/Compat.jl/issues/634). This is not really a strong dependency on newer Julia versions ;).